### DISCUSSION TITLE
有没有交流群？

### DISCUSSION THREAD

**Discussion by fjutnetcxloong** [baidu/amis#6519] [Q&A]
QQ群，微信群什么的有吗？

**Comment by bamzc** [comment]
我的微信bam_zc，我也正在找。

**Reply by hejun200613224** [reply]
有找到组织吗，亲

**Reply by bamzc** [reply]
> 有找到组织吗，亲

自己弄了个群，没几个人

**Reply by bamzc** [reply]
加我微信哟，人数正在越来越壮大了。期待你们的加入，bam_zc

**Comment by fjutnetcxloong** [comment]
717791727或者721182449




楚小龙
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "baidu/amis"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月8日(星期一) 下午2:02
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [baidu/amis] 有没有交流群？ (Discussion #6519)





 
我的微信bam_zc，我也正在找。

**Comment by elarbor** [comment]
[discussions](https://github.com/baidu/amis/discussions/)就挺好的嘛

**Comment by bamzc** [comment]
加我微信哟，人数正在越来越壮大了。期待你们的加入，bam_zc
